tests/commands/run_test.py

```diff
@@ -187,6 +187,13 @@ def test_global_exclude(cap_out, store, tempdir_factory):
             },
             b'All changes made by hooks:',
         ),
+        (
+            {
+                'show_diff_on_failure': True,
+                'color': 'auto',
```

Comment:
so as above, this isn't a valid option here unfortunately

Reply:
But the tests pass :)

Reply:
yeahhhh the tests pass because we ignore the return code of `git diff` -- it actually ends up printing something like this:

```console
$ git diff --color=True
error: option `color' expects "always", "auto", or "never"
```